python/ruff-ecosystem/ruff_ecosystem/check.py

```diff
@@ -99,14 +99,6 @@ def markdown_check_result(result: Result) -> str:
         )
     lines.append("")
 
-    # Limit the number of items displayed per rule to between 5 and 50
-    max_display_per_rule = max(
-        5,
-        # Calculate the number of affected rules that we would display to increase
-        # the maximum if there are less rules affected
-        50 // total_affected_rules,
```

Comment:
Division by zero can occur here if there are only errors (i.e. no rule changs) this commit was meant to be in #8223 but I missed it.